pkg/kubelet/apis/config/types.go

```diff
@@ -568,3 +573,9 @@ type MemoryReservation struct {
 	NumaNode int32
 	Limits   v1.ResourceList
 }
+
+// PodPriorityShutdownGracePeriod specifies the shutdown grace period for Pods based priorities
```

Comment:
specifies the shutdown grace period for Pods based on their associated priority class value.